contributing_to_docs/doc_guidelines.adoc

```diff
@@ -52,7 +52,18 @@ toc::[]                                                         <7>
 [NOTE]
 ====
 * The `{product-title}` and `{product-version}` common attributes are not defined in the `_attributes/common-attributes.adoc` file. Those attributes are pulled by AsciiBinder from the distro mapping definitions in the https://github.com/openshift/openshift-docs/blob/main/_distro_map.yml[_distro_map.yml] file. See xref:product-name-and-version[Product title and version] and xref:attribute-files[attribute files] for more information on this topic.
-* If you use a variable in the title of the first assembly in a section, move the include attributes directive above the title in this assembly. Otherwise, the variable will not render correctly on access.redhat.com.
+* If you use a variable in the title of the first assembly in a section, move the include attributes directive above the anchor ID in this assembly. Otherwise, the variable will not render correctly on our documentation portals. For example:
+
+[literal]
+....
+ :_mod-docs-content-type: ASSEMBLY
+ include::_attributes/common-attributes.adoc[]
+ [id="installing-ibm-cloud-private"]
+ = Installing a private cluster on {ibm-cloud-title}
+ :context: installing-ibm-cloud-private
+
+toc::[]
+....
```

Comment:
Again this is something I'm not familiar with since this isn't part of our docs, but should this be wrapped in `----` like the example above on lines 38-46? 
```suggestion
----
 :_mod-docs-content-type: ASSEMBLY
 include::_attributes/common-attributes.adoc[]
 [id="installing-ibm-cloud-private"]
 = Installing a private cluster on {ibm-cloud-title}
 :context: installing-ibm-cloud-private

toc::[]
----
```

I'm unsure so feel free to disregard if there's a reason this is formatted this way

Reply:
Unfortunetly, the build will fail if I go with:

[source, text]
----
A
B
C
D
----

I could try and indent items inside the delimited space if that would be OK with you? @skopacz1

Reply:
The Asciidoc interpreter picked up "include::_attributes/common-attributes.adoc[]" as a valid include. I tried various ways to mask the include (escape characters, subs, and so on), but the literal attribute and single indentation worked.

Reply:
Have you tried it without the `[source,text]` line in it? The [example](https://github.com/openshift/openshift-docs/pull/76286/files#diff-719cf3b36e5cfdc45393ebde8ff09fd9a56c239b5671fe1cea5b7f364d647125R38-R53) at the top of this metadata section only has `----` in it, so maybe that has something to do with it. If you tried that and it still doesn't work, and the current text you have works, I think it might be fine to keep it.

Reply:
What fixes this is adding a space before each line entry so that the AsciiDoc processor ignores the lines. I added the [source,text] include and things look good.